### DISCUSSION TITLE
webpage remains blank

### DISCUSSION THREAD

**Discussion by xin007-kong** [ricklamers/gpt-code-ui#13] [Q&A]
​What I expected was to be able to see the ui as indicated by the instructions.  Following the instruction, I first pip install gpt-code-ui and then typing gptcode in the terminal with the 127.0.0.1:8080 automatically starting and the webpage opened.  However, the webpage remains blank for a long time.

**Comment by ricklamers** [comment]
If you install a local package `pip install -e .` you should run `make compile_frontend`.

Another aspect you might run into is that the webpage is opened before the Flask app is actually ready to serve the request. In which case a reload (Ctrl/Command + R) should suffice.

**Comment by MATHEUSFELIX** [comment]
In main.py go to server_static funcion and just add  ,mimetype="application/Javascript" as a parameter on send_from_directory call
